core/domain/draft_upgrade_services.py

```diff
@@ -129,10 +129,7 @@ def _convert_html_in_draft_change_list(
             list(ExplorationChange). The converted draft_change_list.
         """
         for i, change in enumerate(draft_change_list):
-            # Here, we are using Any type because new_value can contain values
-            # of type Dict, OutcomeDict, str, list, HintDict and other types
-            # too.
-            new_value: Any = None
+            new_value = None
```

Comment:
Why does this need to be defined at all here (since it is defined almost immediately after)?

Reply:
Removed!